chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -1383,6 +1383,13 @@ pdfReader.promptTransferFromPDF.text = Annotations stored in the PDF file will b
 pdfReader.promptTransferToPDF.title = Store Annotations in File
 pdfReader.promptTransferToPDF.text = Annotations will be transferred to the PDF file and will no longer be editable in %S.
 pdfReader.promptPasswordProtected = The operation is not supported for password-protected PDF files.
+pdfReader.promptDeletePages.title = Delete Pages
+pdfReader.promptDeletePages.text = Delete selected pages from the PDF file
+pdfReader.promptRotatePages.title = Rotate Pages
+pdfReader.promptRotatePages.text = Rotate selected pages in the PDF file
```

Comment:
- I don't think there's any need to prompt before rotating. You're unlikely to do that by mistake, and if you do you can just rotate back. We're not losing any quality through these operations, are we?
- Add question mark for delete prompt
- A little alarming to say "selected pages" when only one page is selected, particularly for deletes — if I were a user I would be worried it was deleting more than just my one selected page. I think we'll want to pass the number of pages here and use plural forms. See `pane.tagSelector.deleteAutomatic.message` for an example.

Reply:
> We're not losing any quality through these operations, are we?

No, we don't.